.github/workflows/webpack.yml

```diff
@@ -0,0 +1,24 @@
+name: sentryConfig
+
+on:
+  release:
+    types: [published, prereleased]
+
+
+jobs:
+  createSentryRelease:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@master
+      - name: Create a Sentry.io release
+        uses: tclindner/sentry-releases-action@v1.0.0
```

Comment:
I would vote for using an official action instead: https://github.com/marketplace/actions/sentry-release